version/version.go

```diff
@@ -0,0 +1,42 @@
+package version
+
+import (
+	"fmt"
+	"strings"
+)
+
+var (
+	// The git commit that was compiled. These will be filled in by the compiler.
+	GitCommit string
+
+	// The main version number that is being run at the moment.
+	//
+	// Version must conform to the format expected by
+	// github.com/hashicorp/go-version for tests to work.
+	Version = "0.8.2"
+
+	// A pre-release marker for the version. If this is "" (empty string)
+	// then it means that it is a final release. Otherwise, this is a pre-release
+	// such as "dev" (in development), "beta", "rc1", etc.
+	VersionPrerelease = "dev"
+)
+
+// GetHumanVersion composes the parts of the version in a way that's suitable
+// for displaying to humans.
+func GetHumanVersion() string {
+	version := Version
+	release := VersionPrerelease
+
+	if release != "" {
+		if !strings.HasSuffix(version, "-"+release) {
+			// if we tagged a prerelease version then the release is in the version already
+			version += fmt.Sprintf("-%s", release)
```

Comment:
@mkeeler this will change the current delimiter for prerelease versions from `.` (https://github.com/hashicorp/serf/pull/599/files#diff-f249ac3c4d1639887402e5e0d7326f0bL29) to `-` to be consistent with other projects.